packages/e2e-tests/testUtils.ts

```diff
@@ -204,6 +204,29 @@ export async function editFileAndWaitForHmrComplete(file, replacer, fileUpdateTo
 	}
 }
 
+export async function editFileAndWaitForBuildWatchComplete(file, replacer) {
+	const newContent = editFile(file, replacer);
+
+	try {
+		await waitForBuildWatchAndPageReload(hmrUpdateTimeout);
+	} catch (e) {
+		const maxTries = isCI && isWin ? 3 : 1;
+		let lastErr;
+		for (let i = 1; i <= maxTries; i++) {
```

Comment:
not sure if we need this, unlike hmr where edits can happen super fast there is a build step in between here so it should work the first time always